recipes/litehtml/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+import os
+import textwrap
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class LitehtmlConan(ConanFile):
+    name = "litehtml"
+    description = "litehtml is the lightweight HTML rendering engine with CSS2/CSS3 support."
+    license = "BSD-3-Clause"
+    topics = ("render engine", "html", "parser")
+    homepage = "https://github.com/litehtml/litehtml"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _min_cppstd(self):
+        return "11"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, self._min_cppstd)
+        if self.options.shared and self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("litehtml must be built as static on windows")
```

Comment:
I think you need to retrigger c3i, e.g. by applying this suggestion.
```suggestion
            raise ConanInvalidConfiguration("litehtml must be built as a static library on windows")
```